packages/jest-preset-default/jest-preset.json

```diff
@@ -23,5 +23,6 @@
 	"transform": {
 		"^.+\\.[jt]sx?$": "<rootDir>/node_modules/babel-jest"
 	},
-	"verbose": true
+	"verbose": true,
+	"reporters": [ "../../../@wordpress/jest-preset-default/scripts/travis-fold-passes-reporter.js" ]
```

Comment:
hmm... is this path right?

Reply:
Yes, `<rootDir>` does not work there in `.json` files for some reason.

Reply:
Did you test it outside of Gutenberg repository? In other places we use `<rootDir>`, e.g.:
`<rootDir>/node_modules/@wordpress/jest-preset-default/scripts/setup-globals.js`

As far as I remember this is the only way to make it work for Jest presets. At least this was the case in the past.

Reply:
huh, I would have thought the path would be relative to the `jest-preset.json` file.  I'm just wondering how well/if this will work things consuming this package (other than gutenberg).  So with this being a relative path, would `./scripts/travis-fold-passes-reporter.js` not work?

Reply:
It's relative to the core jest package.

https://github.com/facebook/jest/blob/5177236f31c705f1084822a9bb7b7da976a420f6/packages/jest-core/src/TestScheduler.ts#L311

Reply:
https://github.com/thymikee/jest-preset-angular - it looks like you no longer are forced to us `jest-preset.json` :)

I think I used this preset as a pattern for initial implementation.

Reply:
So we should change it to `.js`?

Reply:
It looks like they migrated to `jest-preset.js` in this PR: https://github.com/thymikee/jest-preset-angular/pull/204. I assume Jest improved the handling for presets.

Reply:
Still doesn't replace `<rootDir>`.

Reply:
@gziolo might be suggesting to resolve it using something like `require.resolve` in JavaScript?

```js
require.resolve( '@wordpress/jest-preset-default/scripts/travis-fold-passes-reporter' )
```

Reply:
We should use `<rootDir>/node_modules/@wordpress/jest-preset-default/scripts/...` like with the other paths in this file, but first we need to update the package so that the new reporter file is present in the directory in `node_modules`.

Reply:
I don't remember :) The way it's implemented at the moment, it won't work when installed from npm for all projects using `wp-scripts test-unit-js`.